external/buildscripts/sources.txt

```diff
@@ -209,8 +209,6 @@ mono/metadata/class.c
 mono/metadata/class-accessors.c
 mono/metadata/cominterop.c
 mono/metadata/console-unity.c
-mono/metadata/console-win32.c
-mono/metadata/console-unix.c
```

Comment:
We will need a change in the MonoSourceFileList.cs file in the IL2CPP repo to correspond to this change as well. These files should be removed from the list of platform-specific files to build.